packages/contract-tests/test/util.js

```diff
@@ -2,7 +2,7 @@ const debug = require("debug")("test:util");
 const fs = require("fs");
 const ganache = require("ganache");
 const Web3 = require("web3");
-const Web3PromiEvent = require("web3-core-promievent");
+const Web3PromiEvent = require("web3-core");
```

Comment:
The name of the `const` should probably be `Web3Core` instead of `Web3PromiEvent`, as the later use of `new Web3PromiEvent.Web3PromiEvent();` doesn't make sense

Reply:
Could also import as you do in `packages/contract/lib/manual-send.js`:

`const { Web3PromiEvent } = require("web3-core");`